mlxtend/evaluate/accuracy.py

```diff
@@ -0,0 +1,57 @@
+# Sebastian Raschka 2014-2019
+# mlxtend Machine Learning Library Extensions
+#
+# A function for generating per class accuracy
+#
+# License: BSD 3 clause
+import numpy as np
+
+
+def _compute_metric(target_temp, predicted_temp):
+    return (target_temp == predicted_temp).sum() / float(target_temp.shape[0])
+
+
+def accuracy_score(y_target, y_predicted, method="binary", pos_label=1):
+    """General accuracy function for supervised learning.
+    Parameters
+    ------------
+    y_target : array-like, shape=[n_values]
+        True class labels or target values.
+    y_predicted : array-like, shape=[n_values]
+        Predicted class labels or target values.
+    method : str, 'standard' by default.
+        The chosen method for accuracy computation.
+        If set to 'standard', computes overall accuracy.
+        If set to 'binary', computes accuracy for class pos_label.
+        If set to 'average', computes average per class accuracy.
+    pos_label : str or int, 1 by default.
+        The class whose accuracy score is to be reported.
+        Used only when method is set to 'binary'
```

Comment:
I suggest changing

    Used only when method is set to 'binary'

to 

    Used only when `method` is set to 'binary'